moabb/datasets/ssvep_mamem.py

```diff
@@ -31,13 +31,20 @@
 # MAMEM2_URL = 'https://ndownloader.figshare.com/articles/3153409/versions/2'
 # MAMEM3_URL = 'https://ndownloader.figshare.com/articles/3413851/versions/1'
 
-MAMEM1_URL = "https://archive.physionet.org/physiobank/database/mssvepdb/dataset1/"  # noqa: E501
-MAMEM2_URL = "https://archive.physionet.org/physiobank/database/mssvepdb/dataset2/"  # noqa: E501
-MAMEM3_URL = "https://archive.physionet.org/physiobank/database/mssvepdb/dataset3/"  # noqa: E501
+MAMEM1_URL = (
+    "https://archive.physionet.org/physiobank/database/mssvepdb/dataset1/"  # noqa: E501
+)
+MAMEM2_URL = (
+    "https://archive.physionet.org/physiobank/database/mssvepdb/dataset2/"  # noqa: E501
+)
+MAMEM3_URL = (
+    "https://archive.physionet.org/physiobank/database/mssvepdb/dataset3/"  # noqa: E501
+)
```

Comment:
By the way... all this constants are not used anywhere in entire project...
Do we need them at all?

Reply:
Agreed, without `# noqa` it fits.
These URL are kept because they point directly to the data. Right now, we use `wfdb` package to download data, but it is an old package and if it is broken, we need a backup strat.